sympy/sets/sets.py

```diff
@@ -911,6 +911,15 @@ def _union(self, other):
 
         See Set._union for docstring
         """
+        if len(other.args)>1:
```

Comment:
You need to check the type of `other` here (I'm assuming it should be `isinstance(other, Complement)`).